requirements/development.txt

```diff
@@ -6,16 +6,16 @@
 #    pip-compile-multi
 #
 -r base.txt
--e file:.
+-e file:///Users/max/code/superset
```

Comment:
```suggestion
-e file:.
```

Reply:
oh yeah that's a `pip-compile-multi` bug AFACT, it shows randomly every 2-4 runs for me... Not sure if it's been reported.